chain/network-primitives/src/config.rs

```diff
@@ -99,7 +99,7 @@ pub struct NetworkConfig {
     /// are satisfied.
     /// This flag should be ALWAYS FALSE. Only set to true for testing purposes.
     pub outbound_disabled: bool,
-    /// Not clear old data, set `true` for archive nodes.
+    /// Whether this is an archival node.
     pub archive: bool,
```

Comment:
Peer-manager doesn't care what being an archival node means. It just forwards this information blindly to connected peers. This description is misleading because it suggests that network module is actually doing some GC, which is not the case.